package.json

```diff
@@ -40,28 +38,25 @@
     "supercluster": "^2.3.0",
     "through2": "^2.0.3",
     "tinyqueue": "^1.1.0",
-    "unassertify": "^2.0.0",
-    "unflowify": "^1.0.0",
-    "vt-pbf": "^3.0.1",
-    "webworkify": "^1.5.0"
+    "vt-pbf": "^3.0.1"
   },
   "devDependencies": {
-    "@mapbox/batfish": "^0.13.3",
+    "@mapbox/batfish": "^1.0.4",
     "@mapbox/mapbox-gl-rtl-text": "^0.1.1",
     "@mapbox/mapbox-gl-test-suite": "file:test/integration",
+    "@std/esm": "^0.21.7",
     "babel-eslint": "^7.0.0",
     "benchmark": "~2.1.0",
-    "browserify": "^14.0.0",
+    "browserify": "^16.1.0",
```

Comment:
No longer necessary?

Reply:
Still needed for a build unit test that confirms that the built file can be used with browserify.